app/services/sync_ndbn_members.rb

```diff
@@ -6,26 +6,31 @@ def self.upload(member_file)
       return [e.message]
     end
 
-    member_entries.flat_map do |member_id, account_name|
-      ndbn_member = NDBNMember.find_or_initialize_by(ndbn_member_id: member_id.to_i)
+    member_entries.flat_map do |row|
+      member_id, account_name = row[:ndbn_member_number], row[:member_name]
+      ndbn_member = NDBNMember.find_or_initialize_by(ndbn_member_id: member_id)
       ndbn_member.account_name = account_name
       ndbn_member.save
 
+      begin
+        Integer(member_id)
+      rescue ArgumentError
+        ndbn_member.errors.add(:ndbn_member_id, "id must be an integer")
+      end
```

Comment:
Probably could just check if `member_id.to_i.zero?`.

Reply:
How about a regex match? Matching 0 would disallow "0" as an org number. Idk if that is a concern.

Reply:
I think we can disallow 0.

Reply:
There isn't going to be an ID of 0.  I'm pretty confident of that.